tensorvision/analyze.py

```diff
@@ -0,0 +1,119 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+"""Trains, evaluates and saves the model network using a queue."""
+# pylint: disable=missing-docstring
```

Comment:
This should not be here. On the one hand, this can be deactivated in your editor. On the other hand, functions should have docstrings.